src/Options/index.js

```diff
@@ -241,6 +242,8 @@ export interface ParseServerOptions {
   playgroundPath: ?string;
   /* Callback when server has started */
   serverStartComplete: ?(error: ?Error) => void;
+  /* Rest representation on Parse.Schema https://docs.parseplatform.org/rest/guide/#adding-a-schema */
```

Comment:
Please remove docs reference from code, the docs should reference the code, rather than the other way around, otherwise we will have a lot of broken links over time.